client/layout/masterbar/notifications.jsx

```diff
@@ -139,7 +139,8 @@ class MasterbarItemNotifications extends Component {
 						}
 					/>
 				</MasterbarItem>
-				<Notifications
+				<AsyncLoad
+					require="notifications"
 					isShowing={ this.props.isNotificationsOpen }
 					checkToggle={ this.checkToggleNotes }
 					setIndicator={ this.setNotesIndicator }
```

Comment:
There should be a `placeholder={ null }` prop, so that the `AsyncLoad` component doesn't render anything while loading. If the placeholder has the default value, there is an empty space on the right side of the "bell" icon.

With this change, the patch works flawlessly for me.